### PR TITLE
Translate docs of experience center

### PR DESCRIPTION
**Pull request by hxcGit**

### CODE REVIEW

File: versioned_docs/version-v0.6.0/installation/openyurt-experience-center/kubeconfig.md [hunk 1]
```diff
@@ -2,35 +2,36 @@
 title: How to use `kubeconfig` to experience OpenYurt capabilities
 ---
 
-> 这份文档将介绍如何使用体验中心中提供的 kubeconfig 体验 OpenYurt 的单元化部署能力。
+> This document will describe how to experience the UnitedDeployment capability of OpenYurt through kubeconfig provided by Experience Center.
```
Comment (rambohe-ch) [inline versioned_docs/version-v0.6.0/installation/openyurt-experience-center/kubeconfig.md:5]: how about use `multi-domain workload manage capability` for `单元化部署能力`?

Comment (hxcGit) [reply versioned_docs/version-v0.6.0/installation/openyurt-experience-center/kubeconfig.md:5]: 您是说仅修改这一个地方还是将该文档中全部的`单元化部署能力`全部修改？

Comment (rambohe-ch) [reply versioned_docs/version-v0.6.0/installation/openyurt-experience-center/kubeconfig.md:5]: all docs need to replace with `multi-domain workload manage capability`

Comment (hxcGit) [reply versioned_docs/version-v0.6.0/installation/openyurt-experience-center/kubeconfig.md]: Thanks for your suggestion and  I have modified it.

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**